source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -500,6 +500,42 @@ def geojson():
 BRAGI_MOCK_BOBETTE_DEPTH_ONE = deepcopy(BRAGI_MOCK_BOBETTE_DEPTH_ZERO)
 BRAGI_MOCK_BOBETTE_DEPTH_TWO = deepcopy(BRAGI_MOCK_BOBETTE_DEPTH_ZERO)
 BRAGI_MOCK_BOBETTE_DEPTH_THREE = deepcopy(BRAGI_MOCK_BOBETTE_DEPTH_ZERO)
+BRAGI_MOCK_BOBETTE_WITH_CHILDREN = deepcopy(BRAGI_MOCK_BOBETTE)
+BRAGI_MOCK_BOBETTE_WITH_CHILDREN["features"][0]["properties"]["geocoding"]["children"] = [
+    {
```

Comment:
(do it only if you feel like it)
Maybe add multiple children could be nice here too.